draft-ietf-httpbis-priority.md

```diff
@@ -205,6 +205,17 @@ Where both endpoints disable HTTP/2 priorities, the client is expected to send
 this scheme's priority signal. Handling of omitted signals is described in
 {{parameters}}.
 
+# Applicability of the Extensible Priority Scheme
+
+The priority scheme defined by this document considers only the prioritization
+of HTTP response data. Clients express priority values to a server, which are
+input into scheduling decisions, see {{client-scheduling}} and
+{{server-scheduling}}.
```

Comment:
Not that I have a good suggestion at this moment, but it seems to me that this text has two issues:
* It contradicts with {{client-scheduling}}, which says that a client might use the priority signal for scheduling "requests."
* Payload of successful CONNECT request is not HTTP response, but I think we are going to cover them in this specification? Assuming that is the case, prioritization is not limited to HTTP messages.

Reply:
Thanks, you're right on both counts. 

Over on https://github.com/httpwg/http-extensions/pull/1544/files, we're trying to accommodate the CONNECT method. So we could borrow some ideas. I'll follow up with a suggestion...

Reply:
Incorporated your simplified suggestion now, so resolving this thread